metadata/src/main/java/edu/unc/lib/dl/util/JobStatusFactory.java

```diff
@@ -161,6 +163,18 @@ public void setTotalCompletion(String jobUUID, int totalClicks) {
         }
     }
 
+    public boolean objectIsIngested(String depositId, String objectId) {
```

Comment:
Could you change the name of the two new methods to use the "Completed" language instead of "Ingested"? And add javadoc to describe what they are for